x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/RoleDescriptorTests.java

```diff
@@ -888,36 +888,40 @@ public static RoleDescriptor randomRoleDescriptor(boolean allowReservedMetadata)
     }
 
     public static RoleDescriptor randomRoleDescriptor(boolean allowReservedMetadata, boolean allowRemoteIndices) {
```

Comment:
Note: I refactored this method so I can reuse part of it to build a random role descriptor which always has `remote_indices`.